tests/lean/run/KyleAlg.lean

```diff
@@ -187,7 +187,7 @@ instance [One α] [One β] : One (α × β) where
     one := (1, 1)
 
 theorem Product.ext : {p q : α × β} → p.1 = q.1 → p.2 = q.2 → p = q
-    | (a, b), (c, d) => by simp_all
```

Comment:
This whole file uses four-space indentation

Reply:
yes, and curly braces and semi-colons.

Do you propose to not change the indentation before `|` in this file or to make the style of this file and `KyleAlgAbbrev` to match the rest of the repo. 
There are currently only 69 occurance of `by \{\n` , one of them seems deliberate (because of the comment "`Same example for curly braces and semicolons aficionados`")

Reply:
I can't speak for the lean4 developers, but I think the most sensible choice is either:
* Leave the file alone (reverting the changes here), adapt the rule to "one indentation level" rather than two spaces
* Make a separate PR to clean up files with four-space indents.